client/apollo/react/src/Form/Checkbox/CardCheckbox/CardCheckboxCommon.tsx

```diff
@@ -1,17 +1,15 @@
-import React, {
-  ComponentPropsWithRef,
-  useId,
-  type ComponentProps,
-  type ComponentType,
-} from "react";
+import React, { useId, type ComponentProps, type ComponentType } from "react";
 import { BREAKPOINT } from "../../../utilities/constants";
 import { getComponentClassName } from "../../../utilities/getComponentClassName";
 import { useIsSmallScreen } from "../../../utilities/hook/useIsSmallScreen";
 import { ItemMessage } from "../../ItemMessage/ItemMessageLF";
 import { CardCheckboxItem, type TCardCheckboxItem } from "./CardCheckboxItem";
 import type { CheckboxComponent, IconComponent } from "./types";
 
-export type CardCheckboxProps = ComponentPropsWithRef<"input"> & {
+export type CardCheckboxProps = Omit<
+  React.InputHTMLAttributes<HTMLInputElement>,
+  "disabled" | "size"
+> & {
```

Comment:
pourquoir retirer la prop `disabled`?

Reply:
Pour respecter le typage de CardCheckboxItem

Reply:
Et pourquoi ne pas prendre les props de `CardCheckboxItem`sans les redéfinir ?
```suggestion
export type CardCheckboxProps = ComponentsPropsWithoutRef<typeof CardCheckboxItem> & {
```

Reply:
j'avais hésité mais je ne voulais pas surchargé les props en plus des props natives d'input : 
```ts
{
  label: ReactNode;
  subtitle?: ReactNode;
  description?: ReactNode;
  hasError?: boolean;
  icon?: string;
}
```

Reply:
C'est corrigé 👍